src/assets/accessibilityStrings.ts

```diff
@@ -54,6 +54,9 @@ const ACCESSIBILITY_STRINGS = {
   noQuestions: 'Geen vragen gevonden',
   noSectionsNearby: 'Geen onderdelen in de buurt',
   noSections: 'Geen onderdelen gevonden',
+
+  noInternetConnection: 'De verbinding met het internet is verbroken',
+  internetConnectionBackOnline: 'De verbinding met het internet is hersteld',
```

Comment:
Zou hier "Internetverbinding verbroken/hersteld" van maken. Denk dat dat fijner is.

Reply:
Als in "Internet verbinding verbroken" en "Internet verbinding hersteld"? Kan IDD wel wat fijner zijn

Reply:
Internetverbinding is 1 woord vgm, maar weet het niet zeker haha. Maar idd zo ja, anders is de talkback wat lang